portfolio/src/main/webapp/style.css

```diff
@@ -58,3 +58,41 @@ body {
   background-color: blue;
   color: orange;
 }
+
+/* Styling the slideshow */
+.slideShowNavigation {
+  background-color: white;
+  color: white;
+  width: 400px;
+}
+
+.slideshowNavigation button {
+  background-color: dodgerblue;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  display: inline-block;
+  font-size: 1em;
+  margin: 20px;  
+  margin-top: 0px;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  text-align: center;
+  text-decoration: none;
+  width: 100px;
+}
+
+#slideshow-content-container img {
```

Comment:
nit: use consistent naming convention for identifiers, either all `camelCase` or `dashed-words`

Reply:
Fixed